helm v2.3.0 segfaults if values.yaml doesn't exist

This had squeaked by in a previous version I was using of helm. After I upgraded to 2.3.0 and ran `helm install` I was seeing this:

```
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0xb2ce0]

goroutine 1 [running]:
panic(0x1278d20, 0xc4200140a0)
	/usr/local/go/src/runtime/panic.go:500 +0x1a1
k8s.io/helm/pkg/chartutil.processImportValues(0xc42069d920, 0xc4206b8700, 0x1, 0x0)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/pkg/chartutil/requirements.go:388 +0xe40
k8s.io/helm/pkg/chartutil.ProcessRequirementsImportValues(0xc42069d920, 0xc4206b8700, 0x0, 0x0)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/pkg/chartutil/requirements.go:397 +0x88
k8s.io/helm/pkg/helm.(*Client).InstallReleaseFromChart(0xc42017b400, 0xc42069d920, 0x142ef8d, 0x7, 0xc42000edc0, 0x7, 0x7, 0x11fa140, 0x12bf101, 0xc42000edc0)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/pkg/helm/client.go:101 +0x16d
k8s.io/helm/pkg/helm.(*Client).InstallRelease(0xc42017b400, 0xc42014ca40, 0x3d, 0x142ef8d, 0x7, 0xc42000edc0, 0x7, 0x7, 0x6d02e, 0xc420400e60, ...)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/pkg/helm/client.go:75 +0xa2
main.(*installCmd).run(0xc420421380, 0x0, 0x1d93a00)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/cmd/helm/install.go:220 +0x3e6
main.newInstallCmd.func1(0xc42040f440, 0xc4200b8580, 0x1, 0xb, 0x0, 0x0)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/cmd/helm/install.go:158 +0x157
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).execute(0xc42040f440, 0xc4200b84d0, 0xb, 0xb, 0xc42040f440, 0xc4200b84d0)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:599 +0x234
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420138900, 0xc4203d86c0, 0xc4203d9440, 0xc4203d9680)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:689 +0x367
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420138900, 0xc420030010, 0xc420138900)
	/home/ubuntu/.go_workspace/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
	/home/ubuntu/.go_workspace/src/k8s.io/helm/cmd/helm/helm.go:170 +0x47
```

`helm lint` told me that values.yaml was missing. I touched that file, and the segfault went away.

I get a slightly differ beginning of a failure that sounds similar:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x1280741]

goroutine 1 [running]:
k8s.io/helm/pkg/chartutil.processImportValues(0xc4203d63c0, 0xc42068b6e0, 0x0, 0x0)
	/private/tmp/kubernetes-helm-20170407-29018-4jwl5u/src/k8s.io/helm/pkg/chartutil/requirements.go:388 +0xea1
```
I _do_ have a _values.yaml_ file in all of my charts, so I suspect the cause is also different.

Now I am getting a major failure:

```console
⇒  helm install . --dry-run
2017/04/12 11:50:29 Warning: ImportValues missing table: no table named "foo" (map[global:map[]])
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x127bbd1]

goroutine 1 [running]:
k8s.io/helm/pkg/chartutil.processImportValues(0xc42064b380, 0xc420185f00, 0x1, 0x0)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/pkg/chartutil/requirements.go:388 +0xea1
k8s.io/helm/pkg/chartutil.ProcessRequirementsImportValues(0xc42064b380, 0xc420185f00, 0x0, 0x0)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/pkg/chartutil/requirements.go:397 +0x88
k8s.io/helm/pkg/helm.(*Client).InstallReleaseFromChart(0xc420726a00, 0xc42064b380, 0x294bbf5, 0x7, 0xc4206a8f00, 0x7, 0x7, 0xc420657830, 0x0, 0x0)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/pkg/helm/client.go:101 +0x16c
main.(*installCmd).run(0xc420410a90, 0x0, 0x3558980)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/cmd/helm/install.go:223 +0x33f
main.newInstallCmd.func1(0xc42021d440, 0xc4202850e0, 0x1, 0x2, 0x0, 0x0)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/cmd/helm/install.go:158 +0x167
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).execute(0xc42021d440, 0xc420284f80, 0x2, 0x2, 0xc42021d440, 0xc420284f80)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:599 +0x3e9
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc420441200, 0xc420290d80, 0xc4202b8fc0, 0xc4202b9680)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:689 +0x339
k8s.io/helm/vendor/github.com/spf13/cobra.(*Command).Execute(0xc420441200, 0xc42000e018, 0xc420441200)
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/vendor/github.com/spf13/cobra/command.go:648 +0x2b
main.main()
	/Users/mattbutcher/Code/Go/src/k8s.io/helm/cmd/helm/helm.go:168 +0x47
```

Thanks for that pointer.

--- a/pkg/chartutil/requirements.go
+++ b/pkg/chartutil/requirements.go
@@ -384,8 +384,9 @@ func processImportValues(c *chart.Chart, v *chart.Config) error {
 	if err != nil {
 		return err
 	}
+
 	// set the new values
-	c.Values.Raw = string(y)
+	c.Values = &chart.Config{Raw: string(y)}
 
 	return nil
 }

